go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/pkg/errors
```

Comment:
There are an opened PR #203 about go.mod and It has some comments, and It's seems that a complicated topic, maybe with the xerrors package would be necessary, we'll check it.

Reply:
If we don't want to add go.mod and don't use other dependencies, I can copy xerrors codes for <go1.13

Reply:
I will wait for your option and decide to add go.mod or not

Reply:
It would be nice if we could avoid the `xerrors` package, to avoid having any dependencies at all, but I don’t think that would be possible without copy-pasting in the relevant code from `golang.org/x/xerrors` which is also not a good idea.

But this `go.mod` would need to be appropriate for both `go1.13' builds as well as `go1.12` builds, which makes the issue even more difficult.

Reply:
For me the idea of including the xerrors package or other external dependency not sounds so well, but the alternative are maybe even worse... we could create our own implementation, but we need to check always the compatibility with standard library and check that the behaviour is the same... so it's a problem in a future.

I think that it's a great idea have these two methods, because the final user only will need to import one errors library, and for the migration to standard library could be more easy for them. But I don't know which could be the better solution.

Maybe @davecheney have another idea, or he think that these features are not necessary on this package.

Reply:
You’re not wrong. Putting wrappers of these functions in this package allows it to continue being a drop-in replacement for the main `errors` package.

An interesting idea is that we could just not import the functions unless it’s go1.13 or higher. If you’re using these functions with a lower version, then the standard errors package won’t have these functions either. 🤷‍♀ 

I think we are in a situation, though, where there are no good answers.

Reply:
So you propose do a simple file like `errors_113.go`:

```go
// +build go1.13

package errors

import (
	"errors"
)

func Is(err, target error) bool {
	return errors.Is(err, target)
}

func As(err error, target interface{}) bool {
	return errors.As(err, target)
}
```

And its respective test of course, and only enable this features on the pkg/errors that it will compiles on go1.13, right?

So, I think that is a very interesting point, so if the users only continue using this library for stack trace, and the standard library get to implement it, they will can migrate it easily.  And for our part there not any breaking changes.

Reply:
I don't think only add `Is` `As` `Unwrap` for >=go1.13 is a good idea. Those functions is a good helper to help us handle error, but <go1.13 users who use pkg/errors can't enjoy them. With those functions they can keep using pkg/errors and enjoy those functions

Reply:
Users who use `errors` with <go1.13 still need to include `xerrors` to get the functions anyways.

Reply:
@Sherlock-Holo I think that philosophy of this library always has been add the lacks of the standard library have, and right now only is add the stack trace. So the users could right now, need that library allow a easily migration between pkg/errors and standard library, nothing more.

The actual users can continue using `xerrors` in previous versions or some of the forks of this package that including the `xerrors` inside for use it with previous version.

IMHO think that is a good idea add the support only to the >=go1.13 to the methods, `Is` and `As`

Reply:
ok, let's only support `Is` `As` `Unwrap` for >=go1.13